mmseg/core/optimizers/layer_decay_optimizer_constructor.py

```diff
@@ -110,14 +135,22 @@ def add_params(self, params, module):
             else:
                 group_name = 'decay'
                 this_weight_decay = weight_decay
-
-            if decay_type == 'layer_wise':
-                layer_id = get_num_layer_layer_wise(
-                    name, self.paramwise_cfg.get('num_layers'))
-                logger.info(f'set param {name} as id {layer_id}')
+            if 'layer_wise' in decay_type:
+                if 'ConvNeXt' in module.__class__.__name__:
+                    layer_id = get_num_layer_for_convnext(
```

Comment:
Can the `get_num_layer` and `get_num_stage` functions become methods of the backbone class? In this way, we do not need to judge the class name here.